codegen/lib/objc-codegen.js

```diff
@@ -0,0 +1,280 @@
+  // override possibly inherited values
+  reverseMethod.deprecated = false;
+
+  var reverseModel = findModelByName(models, targetClass);
+  reverseModel.methods.push(reverseMethod);
+  if(reverseMethod.name.match(/create/)){
+    var createMany = Object.create(reverseMethod);
+    createMany.name = createMany.name.replace(
+      /create/,
+      'createMany'
+    );
+    createMany.internal = createMany.internal.replace(
+      /create/,
+      'createMany'
+    );
+    createMany.isReturningArray = function() { return true; };
+    reverseModel.methods.push(createMany);
+  }
+
+  var scopeMethod = Object.create(method);
+  scopeMethod.name = reverseName;
+  // override possibly inherited values
+  scopeMethod.deprecated = false;
+  scopeMethod.internal = false;
+  modelClass.scopes[scopeName].methods[apiName] = scopeMethod;
+  if(scopeMethod.name.match(/create/)){
+    var scopeCreateMany = Object.create(scopeMethod);
+    scopeCreateMany.name = scopeCreateMany.name.replace(
+      /create/,
+      'createMany'
+    );
+    scopeCreateMany.isReturningArray = function() { return true; };
+    apiName = apiName.replace(/create/, 'createMany');
+    modelClass.scopes[scopeName].methods[apiName] = scopeCreateMany;
+  }
+}
+
+function findModelByName(models, name) {
+  for (var n in models) {
+    if (n.toLowerCase() == name.toLowerCase())
+      return models[n];
+  }
+}
+
+function addObjCNames(models, modelPrefix) {
+  for (var modelName in models) {
+    var meta = models[modelName];
+    meta.objcModelName = modelPrefix + modelName[0].toUpperCase() + modelName.slice(1);
+    if (meta.baseModel === 'Model' || meta.baseModel === 'PersistedModel') {
+      meta.objcBaseModel = 'LB' + meta.baseModel;
+    } else {
+      console.error('unknown baseModel name: ' + meta.baseModel); // FIXME
+    }
+
+    meta.objcParams = [];
+    for (var param in meta.params) {
+      var type = meta.params[param].type.name;
+      console.log('param type: ' + meta.params[param].type.name); // FIXME
+      if (type === 'String') {
+        meta.params[param].type.objcName = '(nonatomic, copy) NSString *';
+      } else if (type === 'Number') {
+        meta.params[param].type.objcName = 'long ';
+      } else if (type === 'Boolean') {
+        meta.params[param].type.objcName = 'BOOL ';
+      } else if (typeof type === 'undefined') { // FIXME -- must be an array
+        meta.params[param].type.objcName = '(nonatomic) NSArray *';
+      } else {
+        console.error('ERROR: unknown param type: ' + meta.params[param]); // FIXME
+      }
+    }
+  }
+}
+
+function readTemplate(filename) {
+  var ret = fs.readFileSync(
+    require.resolve(filename),
+    { encoding: 'utf-8' }
+  );
+  return ret;
+}
+
+function renderContent(template, modelMetaInfo) {
+  var script = ejs.render(template, {
+    meta: modelMetaInfo
+  });
+  script = ngdocToDox(script);
+
+  return script;
+}
+
+function ngdocToDox(script) {
```

Comment:
I think you should not need this method, it's better to change the template to produce obj-c documentation strings (Is there any widely accepted format? If not, then doxygen may be a good option.)